qiskit/providers/backend_compat.py

```diff
@@ -56,7 +56,10 @@ def convert_to_target(
     # Parse from properties if it exsits
     if properties is not None:
         qubit_properties = qubit_props_list_from_props(properties=properties)
-        target = Target(num_qubits=configuration.n_qubits, qubit_properties=qubit_properties)
+        target = Target(
```

Comment:
Please revert this change.

Reply:
I fixed it at https://github.com/Qiskit/qiskit-terra/pull/9987/commits/3926200fe62bac8bd9be06c06449bc60d3c93224.